core/src/Streamly/Internal/Data/Array/Unboxed/Type.hs

```diff
@@ -367,19 +369,33 @@ toStreamKRev arr =
 -- | Convert an 'Array' into a stream.
 --
 -- /Pre-release/
+{-# INLINE_EARLY read #-}
+read :: (Monad m, Unboxed a) => Array a -> Stream m a
+read = Stream.fromStreamD . toStreamD
+
+-- | Same as 'read'
+--
+{-# DEPRECATED toStream "Please use 'read' instead." #-}
```

Comment:
Remove deprecations from this module?
Move them to the released modules?

Or move these to the deprecated section in the export list?